template/scripts/drupal-vm/config.yml

```diff
@@ -0,0 +1,29 @@
+# Update the hostname to the local development environment hostname
+vagrant_hostname: ${project.machine_name}.localhost
+vagrant_machine_name: ${project.machine_name}
+
+# Set drupal_site_name to the project's human-readable name.
+drupal_site_name: "${project.human_name}"
+
+# Provide the path to the project root to Vagrant.
+vagrant_synced_folders:
+  # Set the local_path for the first synced folder to `../`.
+  - local_path: ../
+    # Set the destination to the Acquia Cloud subscription machine name.
+    destination: ${project.machine_name}
+    type: nfs
+
+# Set drupal_core_path to the `destination` in the synced folder
+# configuration above, plus `/docroot`.
+drupal_core_path: "/var/www/${project.machine_name}/docroot"
+
+# If you build the makefile using BLT's built-in Phing task (recommended),
+# set `build_makefile` to `false`.
+build_makefile: false
+
+drupal_mysql_user: drupal
+drupal_mysql_password: ''
+drupal_mysql_database: drupal
+
+extra_packages:
+  - unzip
```

Comment:
This is no longer required, Drupal VM added it as a hard dependency https://github.com/geerlingguy/drupal-vm/pull/551.